Formula/minipro.rb

```diff
@@ -29,8 +38,6 @@ def install
   test do
     output_minipro = shell_output("#{bin}/minipro 2>&1", 1)
     assert_match "minipro version #{version}", output_minipro
-    output_miniprohex = shell_output("#{bin}/miniprohex 2>&1", 1)
-    assert_match "miniprohex by Al Williams", output_miniprohex
```

Comment:
`miniprohex` got removed in this commit, https://gitlab.com/DavidGriffith/minipro/-/commit/afd45ba6117596c92a15a3b1aaf687218b1a3a73